numpy/core/tests/test_machar.py

```diff
@@ -22,7 +28,7 @@ def test_underlow(self):
             try:
                 self._run_machar_highprec()
             except FloatingPointError as e:
-                self.fail("Caught %s exception, should not have been raised." % e)
+                AssertionError("Caught %s exception, should not have been raised." % e)
```

Comment:
Just to make sure: This is probably the most important error, since it is a silent bug probably (no change in number of test runs).